### DISCUSSION TITLE
How can I create a austenite growing in to pearlite microstructure?

### DISCUSSION THREAD

**Discussion by Henrique-B-Pereira** [idaholab/moose#17212] [Q&A Modules: Phase field] [answered]
Hello,

I'm new to the program. I literally installed and ran the first example yesterday.

I am looking to do a model that involves the growth of austenite from a completely pearlitic microstructure, similar to this example:

![imagem](https://user-images.githubusercontent.com/80003290/109862065-c3385e80-7c3e-11eb-9662-6aa18652b90c.png)

Afterwards, I would like to use the same microstructure generated from the simulations and return with the pearlite transformation.

Is there a tutorial / example that I can take as a basis?

What suggestion can you give me that I can do this simulation?

**Comment by laagesen** [comment, marked as answer]
Hello Henrique,

This is probably best modeled with a single order parameter that distinguishes between the austenite and pearlite phases, and composition variables for each solute atom. A KKS-type model would probably work best:

https://mooseframework.inl.gov/modules/phase_field/MultiPhase/KKS.html

There are some examples given on the above web page. However, they are not going to be physically parameterized for the system you are looking at. Before you start trying to parameterize the model, I would highly suggest that you do some background reading on phase-field modeling if you aren't familiar with the methodology. The following page has a lot of good resources:

https://pages.nist.gov/pfhub/
